package.json

```diff
@@ -156,7 +157,7 @@
     "react-native-level-fs": "^3.0.1",
     "react-native-linear-gradient": "2.5.6",
     "react-native-localize": "^1.4.0",
-    "react-native-modal": "11.1.0",
+    "react-native-modal": "^11.5.6",
```

Comment:
react-native-modal should rather be removed - it wasn't used anywhere. For the top screens (aka modals) we were using a screen on the top of navigation (check RootNavigator). It is less problematic to maintain.

Reply:
no sorry about this we need

Reply:
done with out changes